src/main/java/cafeboard/Board/Board.java

```diff
@@ -0,0 +1,52 @@
+package cafeboard.Board;
+
+import cafeboard.Post.Post;
+import jakarta.persistence.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@EntityListeners(AuditingEntityListener.class)
+@Entity
+public class Board {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+
+    @CreatedDate
+    private LocalDateTime createTime;
+
+    @OneToMany(mappedBy = "board",cascade = CascadeType.REMOVE)
+    private List<Post> PostList;
+
+    public Board() {
```

Comment:
사소한 거긴 한데, 우리가 직접 사용하는 생성자는 아니라서 `public`보다 한 단계 낮춰주는 것이 더 낫긴 합니다.
```suggestion
    protected Board() {
```